src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepositoryImpl.java

```diff
@@ -394,7 +394,8 @@ public PassingRateStatisticsResult summaryPassingRateStatistics(Filter filter, S
 										.join(STATISTICS_FIELD)
 										.on(STATISTICS.STATISTICS_FIELD_ID.eq(STATISTICS_FIELD.SF_ID))
 										.where(STATISTICS.LAUNCH_ID.eq(LAUNCH.ID))
-										.and(STATISTICS_FIELD.NAME.in(executionStatisticsFields)), 0).cast(Double.class)), 2))
+										.and(STATISTICS_FIELD.NAME.in(executionStatisticsFields)
+												.and(STATISTICS_FIELD.NAME.notEqual(EXECUTIONS_TOTAL))), 0).cast(Double.class)), 2))
```

Comment:
can you remove it upper from collection of fields while filternig fields for execution statistics to not create another condition in sql.

Reply:
done